package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "apify-cli",
-    "version": "0.19.0",
+    "version": "0.19.1",
```

Comment:
should be actually 0.19.2 as 0.19.1 is out already (we forgot to bump the version after release again...)